presets/idiomatic.json

```diff
@@ -0,0 +1,7 @@
+{
+    "preset": "jquery",
+    "requireSpacesInsideParentheses": {
+        "all": true,
+        "except": [ "{", "}" ]
```

Comment:
Correct one is 
```js
"except": [ "{", "}", "[", "]", "function" ]
```

And as a said feature, `requireSpacesInsideParentheses` should understand quotes, so case like this:
```js
foo("bar")
```
was acceptable.